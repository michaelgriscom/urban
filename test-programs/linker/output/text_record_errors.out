H:TestRecords:0000:0000:0010:2012151,15,04,26:URBAN-LLM:0019:TestRecords
T:000000:00000000:TestRecords
T:000001:F8000000:TestRecords
T:000002:F8000000:TestRecords
T:000005:F8000000:TestRecords
T:000006:F8000000:TestRecords
T:000007:F8000000:TestRecords
T:000008:F8000000:TestRecords
T:000009:00090000:TestRecords
T:00000C:0C000000:TestRecords
T:0003E8:F8000000:TestRecords
E:000C:000A:TestRecords
