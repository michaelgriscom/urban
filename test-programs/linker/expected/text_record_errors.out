H:TestRecords:0000:0000:0010:2012146,07,46,20:URBAN-LLM:0019:TestRecords
T:000000:00000000:TestRecords
T:000009:00090000:TestRecords
T:00000C:0C000000:TestRecords
E:0005:0003:TestRecords
